### PR TITLE
✨Add Express server

### PR DESCRIPTION
**Pull request by lmachens**
Build guides
https://vitejs.dev/guide/build.html
https://storybook.js.org/docs/react/workflows/publish-storybook#build-storybook-as-a-static-web-application
https://expressjs.com/en/starter/static-files.html

### CODE REVIEW

File: src/server.ts [hunk 1]
```diff
@@ -1,3 +1,15 @@
...
+const app = express();
+
+// Serve storybook production bundle
+app.use('/storybook', express.static('dist/storybook'));
+
+// Serve app production bundle
+app.use(express.static('dist/app'));
+
+app.listen(PORT, () => {
+  console.log(`photoplay app listening at http://localhost:${PORT}`);
```
Comment (phlgr) [inline src/server.ts]:
```suggestion
  console.log(`boilerplate listening at http://localhost:${PORT}`);
```

**Review by phlgr — CHANGES_REQUESTED** [review]

**Review by infoucarsinan — APPROVED** [review]

**Review by MaxMuecket — APPROVED** [review]
lgtm 🏅

**Comment by lajanzen** [comment]
lgtm 🚀

**Review by lajanzen — APPROVED** [review]
lgtm 🚀

**Review by Dev-dfm — APPROVED** [review]
lgtm 👍

**Pull request merged**